experimental/frame_sorter.go

```diff
@@ -0,0 +1,46 @@
+package experimental
+
+import "github.com/grafana/grafana-plugin-sdk-go/data"
+
+// NewFrameSorter - returns a new frameSorter
```

Comment:
Doc comments should be [sentences](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences).
```suggestion
// NewFrameSorter returns a new frameSorter.
```